src/v/coproc/pacemaker_offset_keeper.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+#include "coproc/ntp_context.h"
+#include "coproc/offset_utils.h"
+
+#include <chrono>
+
+using namespace std::chrono_literals;
```

Comment:
define it inside start() which is the only place. no need to pollute the entire namespace. 

or just use the explicit part or scope it to function